.github/workflows/gradle-publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Gradle Package
+
```

Comment:
improve the format of the file, e.g., by adding blank lines, documentation, a,d indentation.